core-test/src/test/scala/org/bitcoins/core/protocol/tlv/LnMessageTest.scala

```diff
@@ -39,7 +39,7 @@ class LnMessageTest extends BitcoinSUnitTest {
       Try(LnMessageFactory(InitTLV)(hex"00100000000001012a030104")).isSuccess)
 
     assert(Try(LnMessageFactory(InitTLV)(hex"00100000000001")).isFailure)
-    assert(Try(LnMessageFactory(InitTLV)(hex"00100000000002012a")).isFailure)
+    assert(Try(LnMessageFactory(InitTLV)(hex"001000000000ca012a")).isFailure)
```

Comment:
Looks like this was copy pasted incorrectly?

https://github.com/lightning/bolts/blob/master/01-messaging.md#appendix-c-message-extension